frontend/src/main/scala/bloop/bsp/RefreshProjects.scala

```diff
@@ -0,0 +1,19 @@
+package bloop.bsp
+
+import bloop.io.AbsolutePath
+import bloop.logging.Logger
+
+import scala.sys.process.ProcessLogger
+import scala.sys.process.Process
+
+object RefreshProjects {
+  def run(workspaceDir: AbsolutePath, command: List[String], logger: Logger): Unit = {
```

Comment:
Just to guarantee the call site this never throws, we should be returning a `Task`, what do you think? 🤔

Reply:
I just replaced this object with call to Forker.run which returns Task